examples/long_ffmpeg.rs

```diff
@@ -0,0 +1,82 @@
+use anyhow::Result;
```

Comment:
mostly copy of existing example, but instead of mp4 it sends infinite test source stream